test/utilities/extensionUtils.ts

```diff
@@ -291,3 +344,98 @@ export async function findVSCodeBinary(): Promise<string> {
 
   return codeBin[0];
 }
+
+export async function verifyExtensionsAreRunning(
+  extensions: ExtensionType[],
+  timeout = VERIFY_EXTENSIONS_TIMEOUT
+) {
+  log('');
+  log(`Starting verifyAllExtensionsAreRunning()...`);
+  if (extensions.length === 0) {
+    log(
+      'verifyExtensionsAreRunning - No extensions to verify, continuing test run w/o extension verification'
+    );
+    return true;
+  }
```

Comment:
Is this new block used to verify that extensions are not present prior to being installed?